src/current/v24.1/migrate-from-geopackage.md

```diff
@@ -7,56 +7,38 @@ docs_area: migrate
 
  CockroachDB supports efficiently storing and querying [spatial data]({% link {{ page.version.version }}/export-spatial-data.md %}).
 
-This page has instructions for migrating data from the [GeoPackage](https://www.geopackage.org/) format into CockroachDB using [`ogr2ogr`](https://gdal.org/programs/ogr2ogr.html) and [`IMPORT`][import].
+This page has instructions for migrating data from the [GeoPackage](https://www.geopackage.org/) format into CockroachDB using [`ogr2ogr`](https://gdal.org/programs/ogr2ogr.html) and [`IMPORT INTO`]({% link {{ page.version.version }}/import-into.md %}).
 
-In the example below we will import a data set with <a href="https://gisdata.mn.gov/dataset/env-mn-springs-inventory" data-proofer-ignore>the locations of natural springs in the state of Minnesota (USA)<a/> that is made available via <a href="https://gisdata.mn.gov" data-proofer-ignore>gisdata.mn.gov</a>.
+In the following example, you will import a data set with the locations of bus stops in the cities of Minneapolis and St. Paul, MN (USA) that is made available via [gisdata.mn.gov](https://gisdata.mn.gov/dataset/us-mn-state-metc-trans-better-bus-stops).
 
-## Before You Begin
+## Before you begin
 
 To follow along with the example below, you will need the following prerequisites:
 
 - CockroachDB [installed]({% link {{ page.version.version }}/install-cockroachdb.md %}) and [running]({% link {{ page.version.version }}/start-a-local-cluster.md %})
 - [`ogr2ogr`](https://gdal.org/programs/ogr2ogr.html)
+    {% include {{page.version.version}}/spatial/ogr2ogr-supported-version.md %}
 - [Python 3](https://www.python.org)
-
-{% include {{page.version.version}}/spatial/ogr2ogr-supported-version.md %}
-
-## Step 1. Download the GeoPackage data
-
-1. Download the zip file containing the spring location data:
-
+- The bus-stop GeoPackage data:
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    curl -o gpkg_env_mn_springs_inventory.zip https://resources.gisdata.mn.gov/pub/gdrs/data/pub/us_mn_state_dnr/env_mn_springs_inventory/gpkg_env_mn_springs_inventory.zip
+    curl -o gpkg_trans_better_bus_stops.zip https://resources.gisdata.mn.gov/pub/gdrs/data/pub/us_mn_state_metc/trans_better_bus_stops/gpkg_trans_better_bus_stops.zip && unzip gpkg_trans_better_bus_stops.zip
     ~~~
 
-1. Unzip the file:
+## Step 1. Convert the GeoPackage data to CSV
```

Comment:
Maybe not for this PR, but this looks similar to a previous file, perhaps they could share an include.

Reply:
Ditto.